### DISCUSSION TITLE
Correct resolveLoader path when node_modules is in another directory?

### DISCUSSION THREAD

**Discussion by Billiam** [webpack/webpack#12414] [Q&A] [answered]
I'm trying to run webpack from a command line app, a wrapper around webpack using the Node api, and I'm having trouble with loader resolution when the app is installed globally. 

When developing locally, I can build a path like:  `resolveLoader: { modules: [path.resolve(__dirname, '../node_modules)] }` and everything's fine, since all of my required loaders are direct project dependencies.

When installing my app globally, yarn will install some of the dependencies  to `~/.config/yarn/global/node_modules/my-app/node_modules` as expected, but most others to `~/.config/yarn/global/node_modules` instead, and these will be unresolvable with the above config.

My understanding of the `modules: ["node_modules]` string value is that it will resolve to parent directories when the required package can't be found, but that providing an absolute (or relative) path instead will not resolve in the same way. (per https://github.com/webpack/webpack/issues/6505#issuecomment-366502560)

What should the value be for resolveLoader.modules so that both `my-app/node_modules` as well as `my-app/../node_modules` (and so on) will be considered, _when working outside of the project directory_?

**Comment by alexander-akait** [comment, marked as answer]
Just setup multiple of `resolveLoader.modules`, you can specify any count of directories

**Reply by Billiam** [reply]
Thanks. I've set the array to 'every path up to the root that ends with `node_modules`', but it felt like I was doing a BadThing.
